aws/resource_aws_api_gateway_stage_test.go

```diff
@@ -26,6 +26,9 @@ func TestAccAWSAPIGatewayStage_basic(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_api_gateway_stage.test", "stage_name", "prod"),
 					resource.TestCheckResourceAttr("aws_api_gateway_stage.test", "cache_cluster_enabled", "true"),
 					resource.TestCheckResourceAttr("aws_api_gateway_stage.test", "cache_cluster_size", "0.5"),
+					resource.TestCheckResourceAttr("aws_api_gateway_stage.test", "canary_settings.0.percent_traffic", "33.33"),
```

Comment:
Can you please move the `canary_settings` testing to its own acceptance test(s)/configuration(s) and leave just a simple check in the basic test?

```go
resource.TestCheckResourceAttr("aws_api_gateway_stage.test", "canary_settings.#", "0"),
```